metron-platform/elasticsearch-shaded/pom.xml

```diff
@@ -99,7 +148,15 @@
                               </excludes>
                             </filter>
                           </filters>  
```

Comment:
Maybe a comment as to why we are doing relocations will help future maintainers

Reply:
Yes, absolutely agreed.